### PR TITLE
Episode section's responsive layout for a laptop and a desktop

### PR DESCRIPTION
**Pull request by shiv-chan**
Hi 👋 

I updated some CSS and implemented a responsive layout for a laptop and a desktop of the "Episode" section.
Please review the code and test it on your browser.

Thanks!

### CODE REVIEW

File: components/AudioPlayer.module.css [hunk 1]
```diff
@@ -250,96 +254,60 @@
...
+		align-items: center;
+		align-self: center;
+	}
+
+	.audioPlayerBar {
+		align-items: center;
+		justify-content: flex-end;
+	}
+
+	.audioPlayerPlaybutton {
```
Comment (Shiho317) [inline components/AudioPlayer.module.css:301]: Hi. I think its better to add 
margin-bottom: 0;

```suggestion
	.audioPlayerPlaybutton {
	        gap: 5rem;
		margin-bottom: 0;
	}
```

**Review by Shiho317 — CHANGES_REQUESTED** [review]
Hi Kaho! 
Thank you for creating responsive design for laptop!
Its good style for me.
I left one comment. Thank you.

**Review by bluwakpa — APPROVED** [review]
Thank you Kaho! Looks good.

File: components/AudioPlayer.module.css (hunk 1 above)
Comment (shiv-chan) [reply components/AudioPlayer.module.css:301]: Thanks, @Shiho317 ! I'll update this part.

**Review by Shiho317 — APPROVED** [review]
Thank you for changing! It looks perfect for me too!

**Pull request merged**